OpenSDS_Volume_Actions_Design.md

```diff
@@ -0,0 +1,92 @@
+# Title
```

Comment:
Update this placeholder to the title for the design document.

If this is only about extending, then that should be called out instead of "volume actions". If there are other actions planned as part of this, then "actions" is appropriate.

So change to something like:

# Design to enable ability to extend volumes

or 

# Design to implement volume actions

Something like that - those are just suggestions.